scripts/pwned-passwords.sh

```diff
@@ -55,9 +42,13 @@ check_pwned_7z() {
   fi
 }
 
-unzip_pwned_passwords() {
-  echo "Unzipping ${pwned_7z}."
-  7z x $pwned_7z -so | head -n $number_of_passwords | cut -d: -f 1 | sort > $pwned_file
+process_pwned_download() {
+  echo "Processing downloaded password hashes..."
+  find $pwned_tmp_directory -type f -exec cat {} + | \
+    sort -n -r -t: -k 2 | \
+    head -n $number_of_passwords | \
+    cut -d: -f 1 | \
+    sort > $pwned_file
 }
 
 check_passwords() {
```

Comment:
so my machine didn't have `sha1sum` , but it does have `shasum -a 1`

Reply:
i feel like it's clearer to have `"password"` there but we could def name it as a variable/comment it